packages/ketcher-core/src/application/render/renderers/BaseMonomerRenderer.ts

```diff
@@ -55,109 +58,109 @@ export abstract class BaseMonomerRenderer extends BaseRenderer {
-      fill = '#FF7A00';
-      stroke = '#FF7A00';
+  public appendAttachmentPoint(AttachmentPointName, number) {
+    let rotation;
+    let position;
+    let x;
+    let y;
+    let length;
+    let y2;
+    let cx;
+    let cy;
+
+    switch (number) {
+      case 0:
+        rotation = 0;
+        position = { x: 0, y: this.bodyHeight / 2 };
+        x = -18;
+        y = -10;
+        break;
+      case 1:
+        rotation = 180;
+        position = { x: this.bodyWidth, y: this.bodyHeight / 2 };
+        x = 5;
+        y = -10;
+        break;
+      case 2:
+        rotation = 90;
+        position = { x: this.bodyWidth / 2, y: 0 };
+        x = -7;
+        y = -22;
+        break;
+      case 3:
+        rotation = 270;
+        position = { x: this.bodyWidth / 2, y: this.bodyHeight };
+        x = -5;
+        y = 30;
+        break;
+      case 4:
+        rotation = 60;
+        position = { x: this.bodyWidth / 5, y: 5 };
+        x = -10;
+        y = -20;
+        y2 = 5;
+        cx = -20;
+        cy = 5;
+        break;
+      default:
+        rotation = 150;
+        position = { x: (this.bodyWidth / 5) * 4, y: 5 };
+        x = 20;
+        y = -18;
+        y2 = 5;
+        cx = -20;
+        cy = 5;
     }
 
-    const attachmentPointElement = rootElement
-      ?.append('g')
-      .attr('transform', `translate(${position.x}, ${position.y})`);
-
-    const attachmentPointRotationGroup = attachmentPointElement
-      .append('g')
-      .attr('transform', `rotate(${rotation})`);
-
-    attachmentPointRotationGroup
-      .append('line')
-      .attr('x1', 0)
-      .attr('y1', 0)
-      .attr('x2', -6)
-      .attr('y2', 0)
-      .attr('stroke', stroke)
-      .attr('stroke-width', '2px');
-
-    attachmentPointRotationGroup
-      .append('circle')
-      .attr('r', 6)
-      .attr('cx', -12)
-      .attr('cy', 0)
-      .attr('stroke', fill === 'white' ? '#0097A8' : 'white')
-      .attr('stroke-width', '1px')
-      .attr('fill', fill);
-
-    return attachmentPointElement;
-  }
-
-  public appendR1AttachmentPoint(
-    rootElement: D3SvgElementSelection<SVGGElement, void>,
-  ) {
-    const attachmentPoint = this.appendAttachmentPoint(
-      rootElement,
-      { x: 0, y: this.bodyHeight / 2 },
-      0,
-      this.monomer.isAttachmentPointPotentiallyUsed('R1'),
-      this.monomer.isAttachmentPointUsed('R1'),
-    );
-    this.appendAttachmentPointLabel(attachmentPoint, 'R1', -18, -10);
-
-    return attachmentPoint;
-  }
-
-  public appendR2AttachmentPoint(
-    rootElement: D3SvgElementSelection<SVGGElement, void>,
-  ) {
-    const attachmentPoint = this.appendAttachmentPoint(
-      rootElement,
-      { x: this.bodyWidth, y: this.bodyHeight / 2 },
-      180,
-      this.monomer.isAttachmentPointPotentiallyUsed('R2'),
-      this.monomer.isAttachmentPointUsed('R2'),
-    );
-    this.appendAttachmentPointLabel(attachmentPoint, 'R2', 5, -10);
-
-    return attachmentPoint;
-  }
+    let atP;
```

Comment:
Please use full name insteasd of abbreviation

Reply:
yes, i will change this code in next PR